LICENSE.md

```diff
@@ -1,13 +1,21 @@
-The MIT License (MIT)
-=====================
+# The MIT License (MIT)
 
-Copyright (c) 2012-2015 `Level` contributors
----------------------------------------
+**Copyright © 2012-present [Contributors](CONTRIBUTORS.md).**
```

Comment:
Need to add `hallmark` manually in this project (because the community metadata is stored here)